### DISCUSSION TITLE
Make the add to cart button and the variations independent

### DISCUSSION THREAD

**Discussion by ghost** [elementor/elementor#21635] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Finally being able to have total freedom of custom

### Describe the solution you'd like

Good morning
it would be good, and great if you did a big update with WC and made the add to cart button and the variations independent.
Like this Elementor plugin: 
[https://www.awesomescreenshot.com/video/15542761?key=cc865b1ee4a07d024d4de9558ef2cdbc](url)

### Describe alternatives you've considered

Go to Prestashop and use the Elementor plugin CREATIV ELEMENTS

**Comment by playeranto** [comment]
Great ideas ! 
It's about time he did this update!
I don't understand why it's still not done. Considering the price we pay

**Comment by beingo24** [comment]
> ### Prerequisites
> * [x]  I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
> * [x]  The feature is still missing in the latest stable version of Elementor ( Elementor Pro )
> 
> ### What problem is your feature request going to solve? Please describe.
> Finally being able to have total freedom of custom
> 
> ### Describe the solution you'd like
> Good morning it would be good, and great if you did a big update with WC and made the add to cart button and the variations independent. Like this Elementor plugin: [https://www.awesomescreenshot.com/video/15542761?key=cc865b1ee4a07d024d4de9558ef2cdbc](url)
> 
> ### Describe alternatives you've considered
> Go to Prestashop and use the Elementor plugin CREATIV ELEMENTS
> 
> ### Additional context
> _No response_

Absolument d'accord ! idée génial
